python/graphstorm/gconstruct/utils.py

```diff
@@ -636,7 +636,8 @@ def cleanup(self):
         # Expected file structure:
         # merged_file file_feature1 file_feature2
         # rmtree will clean up all single feature files as ExtNumpyWrapper does not clean them up
-        self._arr.flush()
+        if self._arr is not None:
```

Comment:
I remember there are two places calling flush

Reply:
ExtNumpyWrapper called it as well. Let me check if this works with all our current test first. I will add it to the ExtNumpyWrapper as well. And add some test to it